### PR TITLE
Added Soulbound Trident

### PR DESCRIPTION
**Pull request by ramdon-person**
## Description
Added the Soulbound trident

## Changes
Added the Soulbound trident

## Testability

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.

### CODE REVIEW

**Comment by ramdon-person** [comment]
I just tested the trident and it works perfectly. 
For example:
you can throw the trident pick it up and its still soulbound.
you can kill yourself (as seen below) and the item stays in your inventory.
image
and you can throw the trident at a mob and it stays soulbound.


Thanks,
ram



image

**Comment by WalshyDev** [comment]
Needs to be added to the research group still

**Comment by ramdon-person** [comment]
Oh of course!

**Comment by ramdon-person** [comment]
I will do that right now

**Comment by ramdon-person** [comment]
It should work now :)

File: src/main/java/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java [hunk 1]
```diff
@@ -1294,6 +1294,10 @@ public boolean onInteract(final Player p, MultiBlock mb, final Block b) {
 		new SoulboundItem(Categories.WEAPONS, (SlimefunItemStack) SlimefunItems.SOULBOUND_SWORD,
 		new ItemStack[] {null, SlimefunItems.ESSENCE_OF_AFTERLIFE, null, null, new ItemStack(Material.DIAMOND_SWORD), null, null, SlimefunItems.ESSENCE_OF_AFTERLIFE, null})
 		.register(true);
+		
+		new ExcludedSoulboundTool(Categories.WEAPONS, (SlimefunItemStack) SlimefunItems.SOULBOUND_TRIDENT,
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java]: Why ExcludedSoulboundTool?

It's not placeable nor a tool.

Comment (ramdon-person) [reply src/main/java/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java]: Sorry about that.

**Comment by ramdon-person** [comment]
Just updated it

**Comment by ramdon-person** [comment]
The other files should be good too. I went through them and checked.

**Pull request merged**